app/routes/room_routes.js

```diff
@@ -12,25 +14,26 @@ const handle404 = customErrors.handle404
 const requireOwnership = customErrors.requireOwnership
 
 router.post('/create-room', requireToken, (req, res, next) => {
-    req.body.room.owner = req.user._id
-    console.log(req.body)
-
-    Room.create(req.body.room)
+  Room.create({ name: req.body.room.name, validUsers: [req.body.user._id], owner: req.body.user._id, messages: [] })
```

Comment:
Working on this from the front end - need to change any instance of req.body.user._id to -> req.body.userId